src/proposals/20240808/SparkEthereum_20240808.t.sol

```diff
@@ -0,0 +1,91 @@
+// SPDX-License-Identifier: AGPL-3.0
+pragma solidity ^0.8.10;
+
+import 'src/SparkTestBase.sol';
+
+interface IRateSource {
+    function getAPR() external view returns (uint256);
+    function decimals() external view returns (uint8);
+}
+
+interface ICappedFallbackRateSource is IRateSource {
+    function source() external view returns (address);
+    function lowerBound() external view returns (uint256);
+    function upperBound() external view returns (uint256);
+    function defaultRate() external view returns (uint256);
+}
+
+interface IKinkedIRM {
+    function RATE_SOURCE() external view returns (address);
+    function getVariableRateSlope1Spread() external view returns (uint256);
+}
+
+contract SparkEthereum_20240808Test is SparkEthereumTestBase {
+
+    address public constant LST_SOURCE              = 0x08669C836F41AEaD03e3EF81a59f3b8e72EC417A;
+    address public constant CAPPED_FALLBACK_WRAPPER = 0xaBc99f366D2bE1f4e5b8DFC0F561a751dd836246;
+
+    address public constant OLD_WETH_INTEREST_RATE_STRATEGY = 0xE27c3f9d35e00ae48144b35DD157F72AaF36c77e;
+    address public constant NEW_WETH_INTEREST_RATE_STRATEGY = 0x6fd32465a23aa0DBaE0D813B7157D8CB2b08Dae4;
+
+    uint256 public constant LST_ORACLE_YIELD = 0.028604597378813033e18;
+
+    constructor() {
+        id = '20240808';
+    }
+
+    function setUp() public {
+        vm.createSelectFork(getChain('mainnet').rpcUrl, 20421368);  // Jul 30, 2024
+        payload = deployPayload();
+
+        loadPoolContext(poolAddressesProviderRegistry.getAddressesProvidersList()[0]);
+    }
+
+    function testOracleDeploy() public {
+        // Not a special number, the APR just happens to be 2.8%
```

Comment:
I'd put this comment, where the hardcoded value is set

Reply:
done